test/MorphoVaultActions.t.sol

```diff
@@ -67,27 +67,6 @@ contract MorphoVaultActionsTest is Test {
         assertApproxEqAbs(IERC4626(morphoVault).balanceOf(address(wallet)), 9713.4779e18, 0.01e18);
```

Comment:
Would be good to also assert that `IERC4626.convertToAssets(shares) == 10_000e6`.

Reply:
Sounds good.  But due to nature of rounding down on share conversion, in this case there will be 1 wei diff.  
So will need to use Approx here too.